src/renderer/assets/styles/components/keyboardsShortcuts.scss

```diff
@@ -52,6 +57,10 @@
                 .keyshortElement_shortcut {
                     color: var(--color--primary);
                     background-color: transparent;
+
+                    #keyEditButt_AnnotationsCreate {
```

Comment:
I need to check the code ... isn't this a very specific keyboard shortcut ID, not a CSS class that applies to all the items?
If not, then the naming "Annotations create" is incorrect but this would be a separate issue :)

Reply:
Yes you're right, my bad ! I'll change that selector